src/server/index.spec.tsx

```diff
@@ -191,8 +195,13 @@ describe(__filename, () => {
         expect(staticPolicy['script-src']).toEqual(undefined);
         expect(staticPolicy['style-src']).toEqual(undefined);
         expect(staticPolicy['worker-src']).toEqual(undefined);
+      });
+
+      it('exposes a default no-op CSP report endpoint', async () => {
+        const server = request(
+          createServer({ env: prodEnv as ServerEnvVars, rootPath }),
```

Comment:
The value of `REACT_APP_USE_INSECURE_PROXY` seems crucial to this test so I suggest setting it explicitly rather than relying on defaults.

Reply:
I double-checked and it appears that the server responds to this endpoint in any case, but you're right, there seems to be a missing test case for when `REACT_APP_USE_INSECURE_PROXY` is set to `true`.